File: src/main/java/cn/misection/filter/service/MoneyService.java
package cn.misection.filter.service;

import org.apache.poi.ss.usermodel.Workbook;

/**
 * @author Military Intelligence 6 root
 * @version 1.0.0
 * @ClassName MoneyService
 * @Description TODO
 * @CreateTime 2022年02月25日 14:38:00
 */
public interface MoneyService {

  Workbook getWorkbook();

  void save();
}

File: KtMapper/src/main/kotlin/cn/misection/relation/service/MoneyService.kt
package cn.misection.relation.service

import org.apache.poi.ss.usermodel.Workbook

/**
 * @ClassName MoneyService
 * @author Military Intelligence 6 root
 * @version 1.0.0
 * @Description TODO
 * @CreateTime 2022年02月26日 15:48:00
 */
interface MoneyService {

  fun getWorkBook(): Workbook

  fun save()
}
